### PR TITLE
Indicator widgets

### PR DESCRIPTION
**Pull request by aviadl**

### CODE REVIEW

File: Widgets/widget-IndicatorActivity.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "id": "indicators-activity",
+  "version": -1,
+  "name": "Indicators Activity",
+  "dataType": "indicators",
+  "widgetType": "line",
+  "size": 1000,
```
Comment (orius123) [inline Widgets/widget-IndicatorActivity.json]: Why the limit. We return just groups.

Comment (aviadl) [reply Widgets/widget-IndicatorActivity.json]: Limits to number of points loaded

File: Widgets/widget-IndicatorManualUsers.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "id": "indicators-manual-user",
+  "version": -1,
+  "name": "Manual Reputation Indicators by User",
+  "dataType": "indicators",
+  "widgetType": "bar",
+  "size": 1000,
```
Comment (orius123) [inline Widgets/widget-IndicatorManualUsers.json]: Can we have more than 1000 users? Maybe we should put a reasonable default in the server instead

File: Widgets/widget-IndicatorActivity.json (hunk 1 above)
Comment (orius123) [reply Widgets/widget-IndicatorActivity.json]: Than we should add it to all line charts

**Comment by aviadl** [comment]
@orius123 fixed user widget size

**Pull request merged**